src/views/partials/landing.js

```diff
@@ -4,8 +4,15 @@
 
 import { getMessage } from '../../utils/fluent.js'
 
-export const landing = data => `
-<section class='hero'>
+export const landing = () => `
+<noscript>
+  <style>
+    .section-transition {
+      opacity: 1;
+    }
+  </style>
+</noscript>
```

Comment:
This works; considering animation sounds good too. It doesn't necessarily deal with slow-loading JS; we could potentially also add a delayed animation in CSS that changes the opacity to 1 after a certain delay, and if the JS loads in time it can disable that and replace it with the actual animation trigger. But also, let's maybe not overthink it, as it's probably pretty much an edge case.

Reply:
I’ve added a more generic way to add entering transitions for the site and am using CSS variables as suggested.

Reply:
… and [9453676](https://github.com/mozilla/blurts-server/pull/2983/commits/94536765ef7ac373b17ed4929b59263335125000) should take care of blocked or JS that is loading too slow.